extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.css

```diff
@@ -23,27 +23,22 @@
   z-index: 2147483642;
 }
 
+.amp-lbg-mask {
```

Comment:
I would also add the other public css classes that get added in JS e.g. `amp-lbv-desc-text` here with an empty body so they are documented in our CSS file as well.